README.md

```diff
@@ -51,17 +51,23 @@ In its current release these are the benchmarks that are executed:
   - `mongodb`: **Deprecated**. [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
   - `mongodb_ycsb`: [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
   - `netperf`: [HP license](http://www.calculate-linux.org/packages/licenses/netperf)
-  - `oldisim`: Apache v2.
-  - `object_storage_service`: Apache v2.
+  - [`oldisim`](https://github.com/GoogleCloudPlatform/oldisim):
+    [Apache v2](https://github.com/GoogleCloudPlatform/oldisim/blob/master/LICENSE.txt)
+  - [`object_storage_service`](https://github.com/intel-cloud/cosbench):
+    [Apache v2](https://github.com/intel-cloud/cosbench/blob/master/LICENSE)
```

Comment:
Actually 1.... tools it uses are gsutil, aws cli, azure cli, and soon the
rackspace cli.  All other code was custom writtlen

On Sat, Dec 19, 2015 at 10:34 AM, Misha Brukman <notifications@github.com>
wrote:

> In README.md
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/763#discussion_r48095415>
> :
>
> > @@ -51,17 +51,23 @@ In its current release these are the benchmarks that are executed:
> >    - `mongodb`: **Deprecated**. [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
> >    - `mongodb_ycsb`: [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
> >    - `netperf`: [HP license](http://www.calculate-linux.org/packages/licenses/netperf)
> > -  - `oldisim`: Apache v2.
> > -  - `object_storage_service`: Apache v2.
> > +  - [`oldisim`](https://github.com/GoogleCloudPlatform/oldisim):
> > +    [Apache v2](https://github.com/GoogleCloudPlatform/oldisim/blob/master/LICENSE.txt)
> > +  - [`object_storage_service`](https://github.com/intel-cloud/cosbench):
> > +    [Apache v2](https://github.com/intel-cloud/cosbench/blob/master/LICENSE)
>
> @cmccoy <https://github.com/cmccoy>, @voellm <https://github.com/voellm>:
> just to clarify, do you mean that:
>
>    1.
>
>    I am linking object_storage_service to the wrong project, so I should
>    remove or modify the *link*, or
>    2.
>
>    I am linking object_storage_service to the right project but you're
>    not using it at all, so I should remove the entire *bullet point*
>
> I am guessing you mean (2) but just wanted to confirm before removing it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/763/files#r48095415>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/
Benchmark like a pro... PerfKitBenchmarker
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker>